vsphere/replace_ds.py

```diff
@@ -1,6 +1,10 @@
 #!/usr/bin/env python3
 import fileinput
 
+# Convert vsphere_nas_datastore to vsphere_cohesity_datastore_destroy in
+# terraform state. After this change in state, the vsphere_nas_datastore
+# will be managed by vsphere_cohesity_datastore_destroy
 with fileinput.FileInput('terraform.tfstate', inplace=True) as file:
-        for line in file:
-                    print(line.replace('vsphere_nas_datastore', 'vsphere_cohesity_datastore_destroy'), end='')
+    for line in file:
+        print(line.replace('vsphere_nas_datastore',
+            'vsphere_cohesity_datastore_destroy'), end='')
```

Comment:
1. I think changing the state file like this is risky. What if this operation is executed partially during restart. There is a potential of corrupting the state file as a result.
2. As we discussed on the call, changing of the state file has to be done as an atomic operation the previous resource or the next resource.

Can you please also ask Anubhav to review this change?